src/framework/theme/components/popover/popover.component.ts

```diff
@@ -46,6 +49,12 @@ export class NbPopoverComponent {
   @Input()
   content: NbPopoverContent;
 
+  /**
+   * Context which will be passed to rendered component instance.
+   * */
+  @Input()
+  context: Object;
```

Comment:
Shouldn't we validate the `context`? For example, clear out `functions` from it?

Reply:
It's just a thought, not sure we really need this.

Reply:
I don't think it's important. Maybe the user will want to pass functions to the content component?